agent/bench-scripts/pbench-gen-iterations

```diff
@@ -207,7 +207,6 @@ sub build_iteration_cmds {
             $cmdline .= " --" . $arg . "=" . $simplex_params{$arg};
         }
         return ($cmdline);
```

Comment:
Along with the white space, a closing brace was lost.

Reply:
Where is the lost closing brace supposed to go?

Reply:
Lost bracket goes in line 208, and it has been restored.